.github/workflows/test.yaml

```diff
@@ -20,6 +23,8 @@ jobs:
     # runs-on: ubuntu-latest
     env:
       REGISTRY: us-east4-docker.pkg.dev/weave-support-367421/weave-images
+    needs: check-which-tests-to-run
+    if: needs.check-which-tests-to-run.outputs.core_integration_tests
```

Comment:
I think this one and the next one should be `if: needs.check-which-tests-to-run.outputs.weave_query_tests`

Reply:
yep